Fix some typos and errors and make max length of array encoding configurable

shonfeder opened the pull request:
Closes informalsystems/apalache#1658

Commit:
Fix incorrectly hard code task name

Commit:
Remove outdated comment

Commit:
Remove leftover output file

Commit:
Fix timestamp format

Should use colons between time segments

Commit:
Make array encoding length configurable

Commit:
Fix default length in workflow

Commit:
Fix method name

Commit:
Improve xml formatting

rodrigo7491 commented on .github/workflows/main.yml:
```diff
@@ -12,6 +12,10 @@ on:
         description: "Limit the benchmark runs to bench suites matching the comma separated names -- leave blank to run all defined experiments"
         required: false
         default: ""
+      array_max_length:
+        description: "Set length of array encoding runs (defaults to 14)"
+        required: false
+        default: ""
```
Isn't the length independent of the selected encoding? If we want to compare two encodings we would need to run both encodings with the same length.

shonfeder replied:
I’m not sure I understand you’re concern: All encoding will run with whatever length is configured here.

rodrigo7491 replied:
That is right. My point is just that this can be used to set the length of runs for encodings other than arrays. I'm just wondering about the name, should have made that clear to begin with.

shonfeder replied:
Oh yes, I see. So by "array encoding runs" I mean "runs that benchmark the array encoding benchmarks" which of course also includes running the oopsla encoding, against which it is being benchmarked.

Any thoughts on how we should clarify the terminology here?

rodrigo7491 replied:
`comparison_max_length` instead of `array_max_length`, or something on those lines? The source of confusion for me was actually the description, because I interpreted it as changing the length for one encoding but not the other.

shonfeder replied:
How this: db44e32?

rodrigo7491 replied:
`ARRAY_MAX_LENGTH` on `main.yml`?

shonfeder replied:
Oops, forgot to save some files after find/replace. Thanks for the catch.

rodrigo7491 reviewed (commented):
Looks good overall, just have a question about `max_length`.

Commit:
Clarify name of length parameter

Commit:
Commit dangling file

rodrigo7491 reviewed (approved):
LGTM

Commit:
Decrease default max length to 8

Because at 14 the test are exceeding our max running time of 6 hours.

Also slightly refactors so that the default length only specified in one
place.
